bigquery2/bigquery_types.bal

```diff
@@ -87,3 +87,20 @@ public type Table record {
     int lastModifiedTime?;
     Field[] fields?;
 };
+
+public type InsertTableData record {
+    string kind = "";
+    InsertError[] insertErrors = [];
+};
+
+public type InsertError record {
+    int index = 0;
```

Comment:
Do we need to have index explicitly here? Isn't it same as the array index of InsertError[] insertErrors?

Reply:
I think we need to have. Here Index parameter points to the index of the row that error applies to.

https://cloud.google.com/bigquery/docs/reference/rest/v2/tabledata/insertAll